pipelines/ocw/ocw-to-hugo.yml

```diff
@@ -0,0 +1,53 @@
+---
+jobs:
+- name: convert-plone-to-hugo
+  plan:
+  - get: ocw-to-hugo
+    trigger: true
+  - get: ocw-json
+    trigger: true
+  - config:
+      image_resource:
+        name: ''
+        source:
+          repository: node
+          tag: 16-bullseye-slim
+        type: docker-image
+      inputs:
+      - name: ocw-to-hugo
+      - name: ocw-json
+      outputs:
+      - name: ocw-markdown
+      platform: linux
+      run:
+        args:
+        - -exc
+        - |
+          yarn install --pure-lockfile
+          node . -i ../ocw-json -o ../ocw-markdown --staticPrefix /courses --strips3
+        dir: ocw-to-hugo
+        path: sh
+    task: convert-json-to-markdown
+  - put: ocw-markdown
+resource_types:
+- name: s3-sync
+  source:
+    repository: mitodl/concourse-s3-sync-resource
+  type: docker-image
+resources:
+- name: ocw-json
+  source:
+    bucket: open-learning-course-data-rc
```

Comment:
Shouldn't this be templated based on the environment, as well as the output bucket below?